tests/integration/records/table_validator.py

```diff
@@ -1,20 +1,56 @@
 import pytz
 import datetime
 import logging
+from sqlalchemy.engine import Engine
```

Comment:
This file, one of the 3 largest in the project, increased in size to 394 lines.  The total size of those files is now 3436 lines (target: 875).  Is this file complex enough to refactor?